app/utils/cookie.js

```diff
@@ -1,22 +1,35 @@
 'use strict'
 
 const session = require('client-sessions')
+const _ = require('lodash')
+const {COOKIE_MAX_AGE, SESSION_ENCRYPTION_KEY} = process.env
+const SESSION_COOKIE_NAME = 'session'
 
 function checkEnv () {
-  if (process.env.SESSION_ENCRYPTION_KEY === undefined) {
+  if (!isValidKey(SESSION_ENCRYPTION_KEY)) {
     throw new Error('cookie encryption key is not set')
   }
-  if (process.env.COOKIE_MAX_AGE === undefined) {
+  if (!isValidKey(COOKIE_MAX_AGE)) {
     throw new Error('cookie max age is not set')
   }
 }
 
+/**
+ * @private
+ *
+ * @param {string} key
+ * @returns {boolean}
+ */
+function isValidKey (key) {
+  return !!key && typeof key === 'string'
```

Comment:
Nice optimisation! Not sure if the future Node.js versions the type of the value can be something not equal to `string`.
The Node.js LTS docs https://nodejs.org/dist/latest-v8.x/docs/api/process.html#process_process_env :
```
Assigning a property on process.env will implicitly convert the value to a string.
```
and the Node.js Current docs https://nodejs.org/dist/latest-v10.x/docs/api/process.html#process_process_env
```
Assigning a property on process.env will implicitly convert the value to a string. This behavior is deprecated. Future versions of Node.js may throw an error when the value is not a string, number, or boolean.
```
As I understand it the value type will be always `string`, but when you pass env variable which is not a string, number, or boolean it will throw an exception.
If this is correct then we should leave this function as it you implemented 👍

Reply:
Thanks Dan :)